src/lib/override_cursor.js

```diff
@@ -0,0 +1,43 @@
+/**
+* Copyright 2012-2017, Plotly, Inc.
+* All rights reserved.
+*
+* This source code is licensed under the MIT license found in the
+* LICENSE file in the root directory of this source tree.
+*/
+
+
+'use strict';
+
+var setCursor = require('./setcursor');
+
+var STASHATTR = 'data-savedcursor';
+
+/*
+ * works with our CSS cursor classes (see css/_cursor.scss)
+ * to override a previous cursor set on d3 single-element selections,
+ * by moving the name of the original cursor to the data-savedcursor attr.
+ * omit cursor to revert to the previously set value.
+ */
+module.exports = function overrideCursor(el3, csr) {
```

Comment:
haha good call - `overrideCursor` was actually broken, though somehow it looked like it was working. Fixed & tested in https://github.com/plotly/plotly.js/pull/1265/commits/ccbffd0fd85efabd17ede2984141c7d47504b449